tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -39,14 +41,18 @@ bool IsRetriable(Status status) {
 
 Status CallWithRetries(const std::function<Status()>& f) {
   int attempts = 0;
+  int delay;
```

Comment:
Since the variable 'delay' is not used outside of the 'while' loop, could you please move it inside the while loop?